### PR TITLE
Plotly modebar appears above modals

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Bug Fix

## Description

Increased z-index base (from `1xxx` to `2xxx`) for all Antd components.

## Related Tickets & Documents

Fixes getredash/redash#3785

### CODE REVIEW

File: client/app/assets/less/ant.less [hunk 1]
```diff
@@ -289,7 +300,7 @@
 }
 
 .ant-popover {
-  z-index: 1000; // make sure it doesn't cover drawer
+  z-index: 2000; // make sure it doesn't cover drawer
```
Comment (ranbena) [inline client/app/assets/less/ant.less]: Not `@zindex-popover`?

Comment (ranbena) [reply client/app/assets/less/ant.less]: Actually, this can be removed altogether?

Comment (kravets-levko) [reply client/app/assets/less/ant.less]: Well, I actually don't know where it's used (tried to find any place but failed). Can you please tell me how to check this?

Comment (ranbena) [reply client/app/assets/less/ant.less]: Screen Shot 2019-05-14 at 7 29 48

Above modal but under drawer (click the "?" icon in the popover to make sure)

Comment (kravets-levko) [reply client/app/assets/less/ant.less]: I was so close :sweat_smile: Will check in a moment. Thanks for pointing on this!

Comment (kravets-levko) [reply client/app/assets/less/ant.less]: @ranbena Fixed. I increased z-index only for help drawer - to place it above all other Antd components.

**Pull request merged**